fe/src/main/java/org/apache/doris/backup/RestoreJob.java

```diff
@@ -806,7 +739,54 @@ private void checkAndPrepareMeta() {
         // No log here, PENDING state restore job will redo this method
         LOG.info("finished to prepare meta and send snapshot tasks, num: {}. {}",
                  batchTask.getTaskNum(), this);
-        return;
+    }
+
+    private boolean backupReplicasNotEqual(PartitionInfo localPartInfo, Partition localPartition, Table localTbl,
```

Comment:
I think the part of `generate file mapping` can be a separated method not including in this method.
Or the name of this method can be confused.

Reply:
can we change the method name to `genFileMappingWhenBackupReplicasEqual`?

Reply:
Sorry for review delay. This name looks good to me.